### PR TITLE
Track upstream v3-release tag, use loopback IP for probes

### LINKED ISSUE #9790: [stable/kiam] Add support for final kiam v3.0 release
kiam v3.0 has been [released](https://github.com/uswitch/kiam/releases/tag/v3.0) 😄.
Add support to the Helm chart.
The following should probably be merged first:
* ~One of https://github.com/helm/charts/pull/9755 or https://github.com/helm/charts/pull/9526~ https://github.com/helm/charts/pull/10353
* https://github.com/helm/charts/pull/9528
* https://github.com/helm/charts/pull/9262
* One of https://github.com/helm/charts/pull/7762 or https://github.com/helm/charts/pull/8942

### PR DESCRIPTION
**Pull request by hakamadare**
#### What this PR does / why we need it:

@ewbankkit 

This addresses two issues that I encountered when using this chart to
deploy kiam v3 on EKS:
1. kiam v3.0 was released 2018-12-06 per
https://github.com/uswitch/kiam/blob/master/CHANGELOG.md, it probably
doesn't make sense to track a RC tag
2. Liveness/readiness probes need to use `127.0.0.1` rather than
`localhost` (see uswitch/kiam#163) because of a TLS change

Signed-off-by: Steve Huff <shuff@vecna.org>

#### Which issue this PR fixes
  - fixes #9790 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by hakamadare** [comment]
@mbarrien good catch, thanks!  fixed now

File: stable/kiam/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: kiam
-version: 2.0.1-rc3
-appVersion: 3.0-rc1
+version: 2.0.1-rc4
```
Comment (ewbankkit) [inline stable/kiam/Chart.yaml]: Could you please change `version` to `2.1.0` as we're no longer using an RC version of the image?

Comment (hakamadare) [reply stable/kiam/Chart.yaml]: @ewbankkit done!  thanks for your attention.

**Comment by ewbankkit** [comment]
@hakamadare Please resolve the conflict on `Chart.yaml`.

**Comment by cheeseandcereal** [comment]
Please note: https://github.com/uswitch/kiam/issues/162#issuecomment-454310998

This upgrade can (and probably will) break compatibility with existing TLS certificates that were previously required to have `localhost` as alternative domain names.

**Comment by hakamadare** [comment]
@ewbankkit done, thanks!

do you have any suggestions for getting this PR reviewed by a maintainer?  i'm a little surprised that what seemed to me like a trivial change has been lingering for two weeks.

**Comment by ewbankkit** [comment]
@hakamadare Frrom what I understand there is maintainer overload, so I can only suggest patience 😄.
The longer term solution is to move the chart to its own repo and use the [Helm Hub](https://github.com/helm/hub/blob/master/README.md) mechanism.

**Comment by cpanato** [comment]
@hakamadare can you please rebase?

**Comment by hakamadare** [comment]
@cpanato thanks for taking a look at this kiam chart PR!  i've addressed the issue that was causing the e2e tests to fail.

**Comment by cpanato** [comment]
@hakamadare please update your branch and resolve the conflict thanks!

**Comment by hakamadare** [comment]
@cpanato fixed, thanks!

is there any reason why you didn't feel comfortable resolving the conflict?  it's exactly the same obvious change as i resolved here (https://github.com/helm/charts/pull/10324/#issuecomment-453681174) and also here (https://github.com/helm/charts/pull/10324/#issuecomment-458125888).  this PR allows edits from maintainers.  i suspect that if any other change is merged that touches this chart, this conflict will be reintroduced, and we will be right back in this situation again.

i'm sorry, i'm travelling right now, and i cannot promise rapid response.

thanks!

**Pull request merged**